lib/api_repo.js

```diff
@@ -316,87 +316,80 @@ ApiRepo.prototype.buildCommonProtoPkgs =
  * @param {function} done the function to run on protoc completion
  */
 ApiRepo.prototype._buildProtos =
-  function _buildProtos(name, version, language, done) {
-    var that = this;
-    /** findOutputs lists the files in the output directory */
-    var findOutputs = function findOutputs(err) {
-      if (err) {
-        console.error('findOutputs:start:err', err);
-        done(err);
-        return;
-      }
-      var outputs = [];
-      var langRoot = path.join(that.outDir, language);
-      var finder = new FindFiles({
-        rootFolder : langRoot,
-        filterFunction : function (_unused, stat) {
-          return stat.isFile();
-        }
-      });
-      var finished = false;
-      finder.on("complete", function() {
-        if (!finished) {
-          done(null, outputs);
-          finished = true;
-        }
-      });
-      finder.on("match", function(path) {
-        var shortPath = path.replace(langRoot + '/', '');
-        outputs.push(shortPath);
-      });
-      finder.on("error", function(err) {
-        if (!finished) {
-          console.error('findOutputs:err', err);
+    function _buildProtos(name, version, language, done) {
+      var that = this;
+      var topDir = this.outDir;
+      var langTopDir = path.join(this.outDir, language);
+
+      /** findOutputs lists the files in the output directory */
+      var findOutputs = function findOutputs(err) {
+        if (err) {
+          console.error('findOutputs:start:err', err);
           done(err);
-          finished = true;
+        } else {
+          var stripRoot = function stripRoot(err, outputs) {
+            if (err) {
+              done(err);
+            } else {
+              var stripped = _.map(outputs, function(x) {
+                return x.replace(langTopDir, '');
+              });
+              done(null, stripped);
+            }
+          };
+          glob.glob("**", {
+            cwd: langTopDir,
+            nodir: true,
+          }, stripRoot);
         }
-      });
-      finder.startSearch();
+      }.bind(this);
```

Comment:
I like `bind(this)` style so this is welcoming, but

- this doesn't fit with the `var that = this;` pattern above, can you remove that?  Or, style change like this might be good to be done in another patch.
- it seems this findOutputs() has no reference to `this`, do you really need to bind?